docs/snippets/web3ToProvider.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Getting Accounts
+ */
+
+// window.web3
+const accounts = web3.eth.accounts;
+
+// window.ethereum
+const accounts = await ethereum.request({ method: 'eth_accounts' });
+
+/**
+ * Sending a Transaction
+ */
+
+// window.web3
+web3.eth.sendTransaction(
+  {
+    to: '0x...',
+    'from': '0x...',
+    value: '0x...',
+    // And so on...
+  },
+  (result) => {
+    // Handle the result
+    console.log(result);
+  }
```

Comment:
It looks like the error handling is missing from this example:

```suggestion
  (error, result) => {
    if (error) {
      return console.error(error)
    }
    // Handle the result
    console.log(result);
  }
```

Reply:
It's missing from our test dapp too :facepalm:

Reply:
😂 👍